pyproject.toml

```diff
@@ -227,7 +227,7 @@ module = [
     "tests.test_builders.test_build_gettext",
     "tests.test_builders.test_build_html",
     "tests.test_builders.test_build_latex",
-    "tests.test_builders.test_build_linkcheck",
+    # "tests.test_builders.test_build_linkcheck",
```

Comment:
i think this can be removed rather than just commented out

Reply:
Taking that discussion to: https://github.com/sphinx-doc/sphinx/pull/12109#discussion_r1534412739 (also has a circular comment-reference back to here)

Reply:
After experimenting, removing or commenting out induce merge conflicts either way. So I'm for removing them (less code to review, and files would anyway be printed out when you mypy on tests).

Reply:
This was a misunderstanding on my behalf; I had noticed the existing commented-out entries, but misinterpreted those as file/module names, and thought that they indicated modules that had already been converted to add type-hinting.

In short: I'll remove this line.